test/apps/uct_info/Makefile.in

```diff
@@ -4,19 +4,24 @@
 # See file LICENSE for terms.
 #
 
-all: uct_info uct_info_static
+all: uct_info uct_info_static ucp_hello_world_static
```

Comment:
would not it be better to define static target in `ucp_hello_world` makefile?
Looks weird that `uct_info` app also builds `ucp_hello_world`

Reply:
yes, since we need to merge this one to fix test issues